### PR TITLE
Update USAGE.md

### PR DESCRIPTION
**Pull request by ciceropablo**
Fix typo.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -4894,7 +4894,7 @@ puts response.headers
 **This endpoint allows you to retrieve the associated link whitelabel for a subuser.**
 
 Link whitelables can be associated with subusers from the parent account. This functionality allows
-subusers to send mail using their parent's linke whitelabels. To associate a link whitelabel, the parent account
+subusers to send mail using their parent's like whitelabels. To associate a link whitelabel, the parent account
```
Comment (mbernier) [inline USAGE.md]: Can you please change this to "Link whitelabels" ?

File: USAGE.md [hunk 2]
```diff
@@ -4916,7 +4916,7 @@ puts response.headers
 **This endpoint allows you to disassociate a link whitelabel from a subuser.**
 
 Link whitelables can be associated with subusers from the parent account. This functionality allows
-subusers to send mail using their parent's linke whitelabels. To associate a link whitelabel, the parent account
+subusers to send mail using their parent's like whitelabels. To associate a link whitelabel, the parent account
```
Comment (mbernier) [inline USAGE.md]: Can you please change this to "Link whitelabels" ?

File: USAGE.md [hunk 3]
```diff
@@ -5013,7 +5013,7 @@ puts response.headers
 **This endpoint allows you to associate a link whitelabel with a subuser account.**
 
 Link whitelables can be associated with subusers from the parent account. This functionality allows
-subusers to send mail using their parent's linke whitelabels. To associate a link whitelabel, the parent account
+subusers to send mail using their parent's like whitelabels. To associate a link whitelabel, the parent account
```
Comment (mbernier) [inline USAGE.md]: Can you please change this to "Link whitelabels" ?

File: USAGE.md (hunk 1 above)
Comment (ciceropablo) [reply USAGE.md]: Done!
Thanks, @mbernier!

File: USAGE.md (hunk 2 above)
Comment (ciceropablo) [reply USAGE.md]: Done!
Thanks, @mbernier!

File: USAGE.md (hunk 3 above)
Comment (ciceropablo) [reply USAGE.md]: Done!
Thanks, @mbernier!

**Comment by thinkingserious** [comment]
Hello @ciceropablo,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by ciceropablo** [comment]
Thanks, @thinkingserious and @mbernier! 👍

**Pull request merged**